<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IResolveDatabaseConflict</class>
 <widget class="QDialog" name="IResolveDatabaseConflict">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database Conflict</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelMsg">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxAll">
     <property name="text">
      <string>Remember choice for all items of this project</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pButClone">
       <property name="text">
        <string>Clone &amp;&amp; Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pButRemote">
       <property name="text">
        <string>Take Remote</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pButLocal">
       <property name="text">
        <string>Force Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pButAbort">
       <property name="text">
        <string>Abort</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
